python/lib/dcos/dcos/subcommand.py

```diff
@@ -567,6 +573,31 @@ def _install_with_binary(
     return None
 
 
+def _rewrite_binary_url(binary_url, dcos_url):
+    """ Rewrite Cosmos binary URLs when needed.
+
+    A rewrite happens when a resource refers to the currently configured
+    DC/OS cluster but uses the wrong scheme. Ideally this "hack" will be
+    removed in the future.
+
+    For more context, see https://jira.mesosphere.com/browse/COPS-3052.
```

Comment:
We should not mention a private ticket in a public repo. Updating [DCOS_OSS-2325](https://jira.mesosphere.com/browse/DCOS_OSS-2325) to add more context and of this comment line to give a public link would be enough.

Reply:
updated